internal/storjql/schema.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storjql
+
+import (
+	"sync"
+
+	"github.com/graphql-go/graphql"
+
+	"storj.io/storj/bootstrap/bootstrapweb"
+	"storj.io/storj/bootstrap/bootstrapweb/bootstrapserver/bootstrapql"
+	"storj.io/storj/satellite/console"
+	"storj.io/storj/satellite/console/consoleweb/consoleql"
+	"storj.io/storj/satellite/mailservice"
+)
+
+// creatingSchemaMutex locks graphql.NewSchema method because it's not thread-safe
+var creatingSchemaMutex sync.Mutex
```

Comment:
I think only expose the mutex here, that way bootstrap and satellite won't end up dependent on each other. At the moment if you try to compile bootstrap it will end up compiling all satellite dependencies into the same binary.